docs/content/integrations/dbt/quickstart.mdx

```diff
@@ -58,6 +58,67 @@ dagster-dbt project scaffold --project-name my-dagster-project --dbt-project-dir
 
 This creates a directory called `my-dagster-project/` inside the current directory. The `my-dagster-project/` directory contains a set of files that define a Dagster project.
 
+</TabItem>
+<TabItem name="Option 2: Use an existing Dagster project">
+
+### Option 2: Use an existing Dagster project
+
+You can use an existing Dagster project to run your dbt project. To do so, you'll need to add some objects using the [dagster-dbt library](/\_apidocs/libraries/dagster-dbt) and update your `Definitions` object. This example assumes that your existing Dagster project includes both `assets.py` and `definitions.py` files.
```

Comment:
We should show the file structure that we expect as a directory tree to be explicit about where we expect files to be.

Reply:
Done in [ce9d1a1](https://github.com/dagster-io/dagster/pull/21817/commits/ce9d1a18fb46e5753d78d93e671b398825a9ec79)